R/lockbox.R

```diff
@@ -68,22 +75,38 @@ lockbox.default <- function(obj) {
   )
 }
 
+reset_to_latest_version <- function(locked_package) {
+  if (locked_package$is_dependency_package) {
+    locked_package$version <- locked_package$latest_version
+  }
+  locked_package$version <- package_version(locked_package$version)
+  locked_package
+}
+
+set_download_dir <- function() {
+  download_dir <- lockbox_download_dir()
```

Comment:
Weird--there already is a function `lockbox_download_dir`. So why do you have a `lockbox_library` function above that seems to do the same thing?